### DISCUSSION TITLE
APIFlask Chatbot for VS Code (GitHub Copilot)

### DISCUSSION THREAD

**Discussion by andrewlayer** [apiflask/apiflask#610] [General]
Hey if any of you guys are using VSCode, my team and I made this extension to help query the APIFlask docs more quickly and accurately!  Its not perfect, but would appreciate feedback of your experience using it.

<a href="https://marketplace.visualstudio.com/items?itemName=buildwithlayer.api-flask-integration-expert-bWPT8" target="_blank" style="text-decoration: none; color: #fff">
  <div style="width: 405px; display: flex; flex-direction: row; align-items: center; gap: 16px; padding: 8px 16px; border: 1px solid #3B3B3B; border-radius: 8px; background-color: #202020; font-family: Manrope, sans-serif;">
    <img src="https://code.visualstudio.com/assets/images/code-stable.png" alt="VS Code Logo"  width="40" height="40" margin-right: 10px;" />
    <div style="display: flex; flex-direction: column">
<a href="https://marketplace.visualstudio.com/items?itemName=buildwithlayer.api-flask-integration-expert-bWPT8" target="_blank" style="text-decoration: none; color: #fff">
      <h3 style="font-size: 18px; font-weight: 500; margin: 0">Download our VS Code Integration Expert</h3>
</a>
    </div>
  </div>
</a>

**Comment by greyli** [comment]
Awesome!
